pkg/doc/signature/suite/jsonwebsignature2020/suite.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package jsonwebsignature2020 implements the JsonWebSignature2020 signature suite
+// for the Linked Data Signatures specification (https://github.com/transmute-industries/lds-jws2020).
+// It uses the RDF Dataset Normalization Algorithm
+// to transform the input document into its canonical form.
+// It uses SHA-256 [RFC6234] as the message digest algorithm.
+// Supported signature algorithms depend on the signer/verifier provided as options to the New().
+// According to the suite specification, signer/verifier must support the following algorithms:
+// kty | crvOrSize | alg
+// OKP | Ed25519   | EdDSA
+// EC  | secp256k1 | ES256K
+// RSA | 2048      | PS256
+// EC  | P-256     | ES256
+// EC  | P-384     | ES384
+// EC  | P-521     | ES512
+package jsonwebsignature2020
+
+import (
+	"crypto/sha256"
+
+	"github.com/piprate/json-gold/ld"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/suite"
+)
+
+// Suite implements jsonWebSignature2020 signature suite
+type Suite struct {
+	suite.SignatureSuite
+}
+
+const (
+	signatureType = "JsonWebSignature2020"
+	format        = "application/n-quads"
+	algorithm     = "URDNA2015"
+)
+
+// New an instance of Linked Data Signatures for JWS suite.
+func New(opts ...suite.Opt) *Suite {
```

Comment:
where are we enforcing the curves, sizes, and algorithms listed in the table above?

Reply:
That's a question. For now, we rely on flexible signers/verifiers solely. It could be a crypto-based signer/verifier.

Reply:
should create issue.